index.html

```diff
@@ -16,123 +23,140 @@
-        </div>  
-          <div class="schedule">
-            <div class="schedule-left">
-           <img class="schedule-images"
-            src="assets/discussion.jpg"
-            alt="Dharma Discussion with Rev Thich Hang Dat and Visitors"
-            height="315px"
-            width="430px"
-           />
-            </br>Friday Sessions: </br>
-            Meditation Session: 6:00-7:00 PM EST </br>
+          The Buddha Blessed Temple offers public meditation instruction,
+          mindfulness training, and Buddhist teachings to Kentuckians of all
+          traditions oriented towards modern life in dealing with stress,
+          anxiety, and depression led by Dr. Rev. Thich Hang Dat, a professor at
+          Indiana University Southeast and University of Houston.
+        </div>
+        <div class="schedule">
+          <div class="schedule-left">
+            <img
+              class="schedule-images"
+              src="assets/discussion.jpg"
+              alt="Dharma Discussion with Rev Thich Hang Dat and Visitors"
+              height="315px"
+              width="430px"
+            />
+            <br />
+            Friday Sessions:
+            <br />
+            Meditation: 6:00-7:00 PM EST
+            <br />
             Buddhist Discussion: 7:00-8:00 PM EST
-            </div>
-            <div class="schedule-right">
-           <img class="schedule-images"
-            src="assets/meditation2.jpg"
-            alt="Thich Giao Dao meditates with guests"
-            height="315px"
-            width="430px"
+          </div>
+          <div class="schedule-right">
+            <img
+              class="schedule-images"
+              src="assets/meditation2.jpg"
+              alt="Thich Giao Dao meditates with guests"
+              height="315px"
+              width="430px"
             />
-            </br>Saturday Session: </br>
-            Meditation Session: 8:00-11:00 AM EST
-            </div>
+            <br />
+            Saturday Session:
+            <br />
+            Meditation: 8:00-11:00 AM EST
+            <br />
+            Recitation: 10:45-11:30 AM EST
           </div>
         </div>
+      </div>
       <div class="locations">
-        <div>
-          <span class="location-title">Tu Viện Vạn Phật Đảnh</span></br>
-          3600 Mathis Road</br>
-          Corydon, IN 47112
-        </div>
-        <div>
-          <span class="location-title">Tu Viện Vạn Phật Đảnh II</span></br>
-          7748 Third Street Road</br>
-          Louisville, KY 40214
-        </div>
-        <div>
-          <span class="location-title">Chùa Phật Giác</span></br>
-          10022 Gaines Road</br>
-          Sugar Land, TX 77498
-        </div>
-        <div>
-          <span class="location-title">Chùa Phật Tâm</span></br>
-          6310 Reega Ave.</br>
-          Egg Harbor Twp, NJ 08234
+        <div class="locations-container">
```

Comment:
Wait, why are all the locations smushed to the top now... Last time it was spread out... just a design comment

Reply:
I think it looks better, but we can change the height of right panel